packages/@snowpack/plugin-build-script/README.md

```diff
@@ -0,0 +1,36 @@
+# @snowpack/plugin-build-script
+
+Plugin that replaces the old `build:*` scripts. Useful when loading a language Snowpack doesn’t support as a first-class citizen.
```

Comment:
This is also our new blessed way to build a file with a CLI. It'll be more future-proof to highlight that workflow and why you'd still want to use this plugin today, and then below that add a "Note: this replaces the old "build:*" scripts used in Snowpack <v2.6".

Reply:
Is there a better description for this? TBH I kinda struggled with a good example of when to use build (as we support a lot of common usecases with official plugins now)

Reply:
Yea, maybe something like:

> A Snowpack plugin to build files in your application using any CLI tool. This plugin passes matching files as input to a custom CLI command, and returns the output response as the build result. This is useful for connecting a custom CLI or when no Snowpack plugin exists for a favorite build tool.

> Note: All Snowpack <v2.6 "build:*" scripts now use this plugin behind the scenes.